style.css

```diff
@@ -1 +1 @@
-body{
	font-family: helvetica, aerial;

	max-width: 90%;
	margin: auto;
	text-emphasis: justify;
}


#menu-img{
	width: 100%;
	max-width: 150px;
	height: auto;
}
#menu-group{
	position: fixed;

	padding-top: 40px;
	float: left;
	width: 20%;
}
#menu-nav{
	text-decoration: none;
	color: #000;

	padding-top: 20px;
	letter-spacing: 3px;
	display: block;
}
#menu-nav:hover{
	color: #555;
}
#menu-nav:active{
	color: #777;
}
#menu-select{
	text-decoration: underline;
	color: #555;

	padding-top: 20px;
	letter-spacing: 3px;
	display: block;
}


.main-card{
	color: #fff;
	background-color: #000;
	border: solid 2px #000;

	margin-top: 30px;
	padding: 5px 20px;
	border-radius: 2px;

	float: right;
	width: 70%;
}
.main-card h1{
	font-size: 25px;
	padding: 0px;
	text-transform: uppercase;
}
.card-img{
	width: 100%;
	max-width: 530px;
	border-radius: 2px;
}
.read-more{
	text-decoration: none;
	color: #fff;
}
.read-more:hover{
	color: #777;
	text-decoration: underline;
}
.read-more:active{
	color: #555;
}


.footer{

}
\ No newline at end of file
+body{
	max-width: 90%;
	margin: auto;
	text-emphasis: justify;
}


#menu-img{
	width: 100%;
	max-width: 150px;
	height: auto;
}
#menu-group{
	position: fixed;

	padding-top: 40px;
	float: left;
	width: 20%;
}
#menu-nav{
	text-decoration: none;
	color: #000;

	padding-top: 20px;
	letter-spacing: 3px;
	display: block;
}
#menu-nav:hover{
	color: #555;
}
#menu-nav:active{
	color: #777;
}
#menu-select{
	text-decoration: underline;
	color: #555;

	padding-top: 20px;
	letter-spacing: 3px;
	display: block;
}


.main-card{
	color: #fff;
	background-color: #000;
	border: solid 2px #000;

	margin-top: 30px;
	padding: 5px 20px;
	border-radius: 2px;

	float: right;
	width: 70%;
}
.card-img{
	width: 100%;
	max-width: 530px;
	border-radius: 2px;
}
```

Comment:
This looks a little funky, might have a look myself to see why we can't make sense of this diff.